deployment-scripts/jobs-deployment/tests/test_deploy_job.py

```diff
@@ -12,9 +12,10 @@
 from .test_s3_utils import put_object
 
 sys.path.insert(0, "../")
-from deploy_job import EmrJobRunner, JinjaTemplateResolver
-from flink_clients import FlinkYarnRunner
-from job_configuration import JobConfiguration, JobConfigurationBuilder
+from deploy_job import EmrJobRunner, JinjaTemplateResolver  # noqa: 402
```

Comment:
Flake8 will complain that these imports are not at the top of the file (because we use `sys.path.insert(...)` before.
If we remove the `sys.path.insert...` line then we will have trouble finding the imported classes. `noqa: 402` ignores that the import is not at the very top of file.

Reply:
For example:
`deployment-scripts/jobs-deployment/tests/test_deploy_job.py:15:1: E402 module level import not at top of file`